### PR TITLE
RM298 and RM614 magazine adjustments

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
The RM298 now spawns with 500-round magazines as default. The 500-round magazines have been nerfed by increasing their weight, volume, and reload time. The 250-round magazines have been nerfed to 8 volume. The RM298 has very mild recoil, so it's description was changed to reference the fact that carrying a massive brick of ammo around isn't conducive to mobile firefights.

Also renamed RMGB500 and removed RM298 magazine_well.

### CODE REVIEW

**Comment by BorkBorkGoesTheCode** [comment]
@Rivet-the-Zombie if you have time, could you look into the default damage bonuses attached to Rivtech guns and rebalance them if needed?

**Comment by Rivet-the-Zombie** [comment]
I don't think they needed to be altered? What's the reasoning behind it?

**Comment by BorkBorkGoesTheCode** [comment]
Nearly all of them (except RM103A, volume 3) have a high damage bonus. If the RM103A has the base barrel length, then the RM99 (volume 3 "ranged_damage": 10) needs to be nerfed.

**Comment by Rivet-the-Zombie** [comment]
Why do you feel that it needs to be reduced? The primary difference between the revolver and semi-auto handgun is that the revolver trades capacity for damage.

File: data/json/items/gun/8x40mm.json [hunk 1]
```diff
@@ -122,8 +122,7 @@
     "reload": 400,
     "barrel_length": 6,
     "valid_mod_locations": [ [ "barrel", 1 ], [ "grip", 1 ], [ "mechanism", 4 ], [ "rail", 1 ], [ "sights", 1 ], [ "stock", 1 ] ],
-    "magazine_well": 6,
```
Comment (Rivet-the-Zombie) [inline data/json/items/gun/8x40mm.json]: Why remove this?

Comment (BorkBorkGoesTheCode) [reply data/json/items/gun/8x40mm.json]: To increase the size of the RM298.

Comment (Rivet-the-Zombie) [reply data/json/items/gun/8x40mm.json]: And why does it need to be larger?

Comment (BorkBorkGoesTheCode) [reply data/json/items/gun/8x40mm.json]: I'm going to remove the mounted flag and re-add the magazine_well

**Pull request merged**